var/spack/repos/builtin/packages/node-js/package.py

```diff
@@ -25,21 +25,15 @@ class NodeJs(Package):
     version("21.7.3", sha256="ce1f61347671ef219d9c2925313d629d3fef98fc8d7f5ef38dd4656f7d0f58e7")
     version("19.2.0", sha256="aac9d1a366fb57d68f4639f9204d1de5d6387656959a97ed929a5ba9e62c033a")
     version("17.9.1", sha256="1102f5e0aafaab8014d19c6c57142caf2ba3ef69d88d7a7f0f82798051796027")
-    version(
```

Comment:
There are some deprecated versions that became undeprecated, is this intentional?

Reply:
see my other comments 😉